implementations/rust/ockam/ockam_command/src/vault/move_vault.rs

```diff
@@ -0,0 +1,56 @@
+use clap::Args;
```

Comment:
Shouldn't the file be called just `move.rs`?

Reply:
It can be called `move` because this is a reserved keyword.